config/testing/client-service-account-rbac.yaml

```diff
@@ -16,6 +16,9 @@ rules:
   - /o2ims-infrastructureMonitoring/api_versions
   - /o2ims-infrastructureCluster/*
   - /o2ims-infrastructureCluster/api_versions
+  - /o2ims-infrastructureArtifacts/*
+  - /o2ims-infrastructureArtifacts/v1/managedInfrastructureTemplates
+  - /o2ims-infrastructureArtifacts/v1/managedInfrastructureTemplates/*
```

Comment:
See my comment in inventory_controller.go regarding the new for us to support the `/api_versions` endpoint.  We can add that in a separate PR, but we should probably just setup the rbac in anticipation of that change now to save the hassle of doing it later.  

```
- /o2ims-infrastructureArtifacts/*
- /o2ims-infrastructureArtifacts/api_versions
```

Reply:
I've modified the test rbac for now and will include the support for the `/api_versions` support in a future PR.